### ISSUE TITLE
Additional commentary / code surrounding assumption-testing of logit

### ISSUE THREAD

**Issue by pmaji** [pmaji/data-science-toolkit#1]
Need to include some additional commentary in the markdown--along with any necessary or pertinent tests--that discuss assumptions necessary when building logits.

**Comment by pmaji** [comment]
@ericctbai has agree to help with this

**Comment by pmaji** [comment]
bump -- still waiting on @ericctbai for this section.

**Comment by ericctbai** [comment]
I nominate Xi Chen. Where’s her fork

**Comment by pmaji** [comment]
Closing this thread out because @ericctbai declined to ultimately contribute.

**Comment by ericctbai** [comment]
Still waiting on @xichen for her fork
